components/search/SearchResults.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+  <!-- search bar div -->
+  <div class="fixed-search-bar">
+    <form class="container" action="/search" method="GET">
+      <div class="position-relative">
+        <!-- Search Icon -->
+        <img class="search-icon" src="/assets/img/search-icon.png" />
+        <div class="vertical-line"></div>
+
+        <!-- Search Input -->
+        <input
+          class="form-control form-control-lg pl-5 search-input"
+          type="text"
+          name="search"
+          placeholder="Search..."
+          autocomplete="off"
+          v-model="searchQuery"
+          @input=""
+          @focus="showSuggestions = true"
+        />
+
+        <!-- Search Suggestions Dropdown -->
+        <ul
+          v-if="showSuggestions && suggestions.length"
```

Comment:
no logics in template, use computed()
```suggestion
          v-if="showSuggestions && suggestions.length"
```